### PR TITLE
Create LIT7074AynatTsella

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
According to our discussion in https://github.com/BetaMasaheft/Documentation/issues/2617.

### CODE REVIEW

**Review by eu-genia — APPROVED** [review]

**Review by nafisa-valieva — APPROVED** [review]
Looks very nice for me! thank you!

File: new/LIT7074AynatTsella.xml [hunk 1]
```diff
@@ -0,0 +1,86 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7074AynatTsella" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">አድኅነኒ፡ እምሕማመ፡ ዓይነ፡ ጽላ፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">ʾAdḫǝnani ʾǝm-ḥǝmāma ʾayna ṣǝlā ...</title>
```
Comment (karljonaskarlsson) [inline new/LIT7074AynatTsella.xml:9]: I would transcribe ʾAdḫǝnanni. And if we interpret Ṣǝlā as a proper noun, then maybe we could capitalize it in the transcription as well?

**Comment by CarstenHoffmannMarburg** [comment]
@eu-genia I have merged this branch three days ago on Friday. 27th. However, I opened a new branch to update the record according to Jonas requests, but I did not find it in Oxygen. Is it lost and necessary to create a new file one or should I wait for it?

**Comment by eu-genia** [comment]
I do not understand. 
Sometimes after a merge a record may need time to get to the app, but it is IMMEDIATELY saved in GitHub repository. 
It is saved to GitHub the moment you commit it to your branch.
When you do the merge your branch is merged with the master branch.
From that moment you MUST sync your local master branch (switch to master and fetch-pull). 
image

**Comment by eu-genia** [comment]
(If you open a new branch without syncing the master, or not from the master but from another branch created before the merge, clearly it won't be there, but as you see it has been on the master branch since 27.09. 11:55)

**Comment by CarstenHoffmannMarburg** [comment]
> From that moment you MUST sync your local master branch (switch to master and fetch-pull).

Yes, I found it by now. I probably have not synced it correctly in my first attempt.

**Pull request merged**